pkg/modules/authorization.go

```diff
@@ -811,6 +814,49 @@ func InstallWithCerts(ctx context.Context, isDeleting bool, op utils.OperatorCon
 	return nil
 }
 
+// getAuthCrdDeploy - apply and deploy authorization crd manifest
+func getAuthCrdDeploy(op utils.OperatorConfig, cr csmv1.ContainerStorageModule) (string, error) {
+	yamlString := ""
+
+	auth, err := getAuthorizationModule(cr)
+	if err != nil {
+		return yamlString, err
+	}
+
+	authCrdPath := fmt.Sprintf("%s/moduleconfig/authorization/%s/%s", op.ConfigDirectory, auth.ConfigVersion, AuthCrds)
```

Comment:
We can call `readConfigFile `function here instead for reading the file

Reply:
Updated